bot/resources/README.md

```diff
@@ -0,0 +1 @@
+Placeholder for external resources for fuzzing on particular platforms.
```

Comment:
I think a Readme should go in `resources/fuchsia`. Not sure if the one here is necessary.

Reply:
moved!